codegen/csg.adoc

```diff
@@ -8,4 +8,193 @@ permalink: /codegen_custom_swagger_gentemplate/
 :RAS: RepreZen API Studio 
```

Comment:
@andylowry , the name of this file is `csg.adoc`.  Was it supposed to be `csg.adoc` ?